ckanext/datajson/blueprint.py

```diff
@@ -377,36 +389,7 @@ def validator():
     return render('datajsonvalidator.html')
 
 
-def _get_ckan_datasets(org=None, with_private=False):
-    n = 500
-    page = 1
-    dataset_list = []
-
-    q = '+capacity:public' if not with_private else '*:*'
-
-    fq = 'dataset_type:dataset'
-    if org:
-        fq += " AND organization:" + org
-
-    while True:
-        search_data_dict = {
-            'q': q,
-            'fq': fq,
-            'sort': 'metadata_modified desc',
-            'rows': n,
-            'start': n * (page - 1),
-        }
-
-        query = p.toolkit.get_action('package_search')({}, search_data_dict)
-        if len(query['results']):
-            dataset_list.extend(query['results'])
-            page += 1
-        else:
-            break
-    return dataset_list
-
-
-datapusher.add_url_rule('/data.json',
+datapusher.add_url_rule('/internal/data.json',
```

Comment:
Pretty sure this should use the config object; pulled in in the `plugin.py` file.

Reply:
Spoke with James. Overriding the default would require an architectural overhaul. This needs to be discussed with upstream.